### ISSUE TITLE
Missing y axis values after chart update

### ISSUE THREAD

**Issue by lticadler** [amcharts/amcharts4#3600]
**Bug description**

Chart with multiple y axis.
Y axis with label text adapters.

Update data where a series has no data.
Update data back to where every series has data.

Sometimes the values in the y axis are missing.
Screenshot from our quality control:
![image](https://user-images.githubusercontent.com/58069494/122762490-a7b09c80-d29d-11eb-847a-1abd3b6569d4.png)

I was able to crate a pen where the behavior is reproducible every time.
https://codepen.io/chrisadler/pen/WNpWVJK?editors=1010

After first update:
![image](https://user-images.githubusercontent.com/58069494/122763373-8a300280-d29e-11eb-8f47-12ed6567b3c7.png)
After second update (no data for series 2):
![image](https://user-images.githubusercontent.com/58069494/122763868-1e01ce80-d29f-11eb-9c92-c296b8a325c2.png)
After third update:
![image](https://user-images.githubusercontent.com/58069494/122763939-32de6200-d29f-11eb-95fe-e0a30dd6b420.png)

**Environment (if applicable)**

 - amCharts 4.10.20
 - Browser and its version: All Browsers

**Comment by martynasma** [comment]
We'll fix it in the next release.

In the interim you can work around it by adding this code:

```JavaScript
  chart.yAxes.each((axis)=>{
      axis.dataItems.each((dataItem)=>{
          dataItem.value = null;
      })
  }) 
```

**Comment by lticadler** [comment]
Thank you 👌 - updated the pen with the given workaround.

**Comment by martynasma** [comment]
Fixed in [4.10.21(https://github.com/amcharts/amcharts4/releases/tag/4.10.21).

## [4.10.21] - 2021-08-08

### Added
- Export: new setting in CSV options: `addBOM` (default `true`).

### Changed
- `userClassName` now supports multiple class names separated by space.
- Exporting CSV will now have add BOM code to exported file.
- Upgrading to `pdfmake 0.2.2` to work around security issues in older version.
- Upgrading to `xlsx 0.17.0` to work around security issues in older version.

### Fixed
- In some cases axis labels could be hidden across multiple data updates.
- Fixed issue with stacked/clustered negative values.
- `MapChart` now respects `home*` zoom/position with externally loaded geodata.
- Sizing/positioning of 3D charts with titles fixed.
- RangeSelector plugin: fixed zoom range resetting issues.
- Issue with inverted radial axis fixed.
- Axis with `snapToSeries` set was snapping to hidden series.
- Using timezone was causing `DateAxis` tooltips to show incorrect time in some cases.
- Issue with label text with a lot of in-line formatting not wrapping properly fixed.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
